config/redirects.js

```diff
@@ -2694,6 +2694,10 @@ module.exports = [
     from: ['/libraries/lock-android/v2/refresh-jwt-tokens','/libraries/lock-android/v1/refresh-jwt-tokens','/libraries/lock-android/refresh-jwt-tokens','/libraries/auth0-android/save-and-refresh-tokens'],
     to: '/libraries/lock-android/lock-android-refresh-jwt'
   },
+  {
+    from: ['/libraries/lock-android/lock-android-delegation'],
+    to: '/libraries/auth0-android/auth0-android-save-and-renew-tokens'
```

Comment:
@avanscoy I've found a better fit for this redirect. Let's use Lock's specific doc for refresh tokens.

```suggestion
    to: '/libraries/lock-android/v2/refresh-jwt-tokens.md'
```